shell/platform/windows/client_wrapper/include/flutter/flutter_view_controller.h

```diff
@@ -61,6 +61,8 @@ class FlutterViewController {
   // loop.
   void ProcessMessages();
 
+  void OnFontChange();
```

Comment:
Needs a declaration comment.

It should also get a TODO to move out of the view controller class once there is an engine class. You can reference #38600